ui/core/src/assets.ethereum.ropsten.json

```diff
@@ -4,52 +4,405 @@
       "symbol": "erowan",
       "decimals": 18,
       "name": "Rowan",
-      "imageUrl": "/images/siflogo.png",
-      "address": "0xA3D31ee81Ec2a898B4CF7A67a0851086e4Da7af3",
+      "imageUrl": "./images/siflogo.png",
+      "address": "xx",
```

Comment:
Is this intentional? Along with the missing addresses? It breaks the UI

Reply:
No, this was a mistake.  The script updates the sandpit file (assets.ethereum.sandpit.json).  I'll update the script to copy assets.ethereum.sandpit.json to assets.ethereum.ropsten.json, but we really should update the UI to use a file that matches the environment (since there will be multiple ropsten deployments, assets.ethereum.ropsten.json won't keep working).